# This message contains the state and outputs of the 
# Decision Making and Monitoring module's state machine

Header header

uint32 state

string object_name
uint32 bin_id
uint32 item_counts
uint32 robot_arm

uint32 localization_mode
uint32 mapping_mode
uint32 object_pose_recognition_mode
uint32 motion_planning_mode
uint32 grasp_planning_mode
